R/use-standalone.R

```diff
@@ -0,0 +1,116 @@
+#' Use a standalone file from another repo
+#'
+#' @description
+#' A "standalone" file implements a minimum set of functionality in such a way
+#' that it can be copied into another package. `use_standalone()` makes it easy
+#' to get such a file into your own repo.
+#'
+#' It always overwrites an existing standalone file of the same name, making
+#' it easy to update previously imported code.
+#'
+#' @inheritParams create_from_github
```

Comment:
This makes the following promise about `repo_spec`:

> `repo_spec`
> A string identifying the GitHub repo in one of these forms:
>
> Plain OWNER/REPO spec
>
> Browser URL, such as "https://github.com/OWNER/REPO"
>
> HTTPS Git URL, such as "https://github.com/OWNER/REPO.git"
>
> SSH Git URL, such as "git@github.com:OWNER/REPO.git"
>
> In the case of a browser, HTTPS, or SSH URL, the host is extracted from the URL. The REPO part will be the name of the new local folder, which is also a project and Git repo.

But no such parsing code is here.

Which also brings up the bigger issue which is that this function doesn't support GHE. In general, our stance is to make stuff like this work for github.com and GHE. I think the toolkit we use elsewhere works well here. I can do it if you want.

Reply:
I switched to `create_from_github()` `repo_spec` and actually parsed it, and added a `host` argument. LMK if there's anything else needed for GHE support.